src/sentry/api/endpoints/project_has_alert_integration_installed.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models import Project
+from sentry.notifications.utils import has_alert_integration
+
+
+class ProjectHasAlertIntegrationInstalled(ProjectEndpoint):
```

Comment:
Can we add `hasAlertIntegrationInstalled` as an `extend` parameter to the `ProjectSerializer`?

Reply:
@mgaeta we could but we only need this data on one page that's not frequently loaded...seems like a waste to do it for every project all the time.

Reply:
hmmm...let me think about it

Reply:
I agree w/ this yeah.

Reply:
I don't think we have anything for `expand` yet for the `ProjectSerializer` but I can go ahead and add it for that endpoint

Reply:
I guess we do `expand` with kwargs like this for a project:

https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/project.py#L152-L171

Reply:
@mgaeta Optional parameters to project serialization is kind of a mess here. I see we are using `include` for optional params on the project details endpoint so I just added it to there and added a TODO.

Reply:
@mgaeta Optional parameters to project serialization is kind of a mess here. I see we are using `include` for optional params on the project details endpoint so I just added it to there and added a TODO.

Reply:
@mgaeta Optional parameters to project serialization is kind of a mess here. I see we are using `include` for optional params on the project details endpoint so I just added it to there and added a TODO.

Reply:
Cool we can refactor the endpoint when we add `expand` to the project serializer. Can you put your variable in an `expand` now so that it won't be a breaking change later?
```python
expand = set(filter(bool, request.GET.get("expand", "").split(",")))
if "hasAlertIntegration" in expand:
  ...
```

Reply:
@mgaeta You mean so we can use `expand` or `include`? Right now we are using `include` for the same purpose

Reply:
`include` is deprecated so your new variable should be in `expand`. In this case, both query parameters would be used.

Reply:
@mgaeta got it, updated so only the new param uses expand